pay-api/src/pay_api/services/payment_account.py

```diff
@@ -564,6 +577,7 @@ def asdict(self, **kwargs):
         # to handle PAD 3 day period..UI needs bank details even if PAD is not activated
         is_future_pad = (self.payment_method == PaymentMethod.DRAWDOWN.value) and (self._is_pad_in_pending_activation())
         show_cfs_details = is_ob_or_pad or is_future_pad
+        d['eft_enable'] = self._dao.eft_enable
```

Comment:
I don't think you need this?

Reply:
I tested with postman, when the eft_enable is false, it doesn't show up. I'm removing `eft_enable = fields.Boolean(...)`

Reply:
That's fine, it's boolean in your model isn't it? FE sets it to false if it's undefined?

Reply:
if it doesn't show up, can just assume it's false? There might be a helper function that removes false and Nones from the payload

Reply:
![image](https://github.com/bcgov/sbc-pay/assets/3484109/ec1e9a26-7516-43d6-a04a-b4104cbfb671)

Reply:
`eft_enable = fields.Boolean(data_key='eft_enable', required=True)`

Reply:
![image](https://github.com/bcgov/sbc-pay/assets/3484109/22719b5e-974b-4831-aa66-99cd4bc3c00e)

Reply:
Might be able to mess with something in here -^